src/amo/components/Category/index.js

```diff
@@ -250,8 +253,20 @@ export class CategoryBase extends React.Component {
 
     const { html } = this.contentForType(addonType);
 
+    const isAddonTheme = isTheme(addonType);
+
+    const themeProps = {};
```

Comment:
This use of `themeProps` feels a bit heavyweight to me. Why not just always pass a `placeholderCount` property to `LandingAddonsCard` and set it to the appropriate value, or maybe just pass an `isTheme` property to `LandingAddonsCard` and allow it to determine the correct value for `placeholderCount`?

Reply:
ok sure, I like the placeholder way : )

Reply:
otherwise, you wouldn't be able to override the placeholder count for a theme, right?

Reply:
i guess you could if you don't pass in isTheme and just use the placeholder count...hmm. I'll try it out : )